StitchCore-iOS/StitchCore-iOS/Services/Internal/ThrowingServiceClientFactory.swift

```diff
@@ -0,0 +1,42 @@
+import StitchCore
+
+/**
+ * A protocol describing a class that can provide clients for a particular named Stitch service.
+ */
+public protocol ThrowingServiceClientFactory {
```

Comment:
[q] where is this coming from? Hadn't I added this already in a previous commit?